azext_iot/iothub/command_map.py

```diff
@@ -33,6 +48,73 @@ def load_iothub_commands(self, _):
         cmd_group.show_command("show", "get_digital_twin")
         cmd_group.command("update", "patch_digital_twin")
 
+    with self.command_group(
+        "iot hub message-endpoint", command_type=iothub_message_endpoint_ops, is_preview=True
+    ) as cmd_group:
+        cmd_group.show_command("show", "message_endpoint_show")
+        cmd_group.command("list", "message_endpoint_list")
+        cmd_group.command(
+            "delete",
+            "message_endpoint_delete",
+            transform=EndpointUpdateResultTransform(self.cli_ctx),
+            confirmation=True
+        )
+
+    with self.command_group(
+        "iot hub message-endpoint create",
+        command_type=iothub_message_endpoint_ops,
+    ) as cmd_group:
+        cmd_group.command(
+            "eventhub",
+            "message_endpoint_create_event_hub",
+            transform=EndpointUpdateResultTransform(self.cli_ctx)
+        )
+        cmd_group.command(
+            "servicebus-queue",
+            "message_endpoint_create_service_bus_queue",
+            transform=EndpointUpdateResultTransform(self.cli_ctx)
+        )
+        cmd_group.command(
+            "servicebus-topic",
+            "message_endpoint_create_service_bus_topic",
+            transform=EndpointUpdateResultTransform(self.cli_ctx)
+        )
+        cmd_group.command(
+            "cosmosdb-collection",
+            "message_endpoint_create_cosmos_db_collection",
+            transform=EndpointUpdateResultTransform(self.cli_ctx),
+            resource_type=ResourceType.MGMT_IOTHUB,
+            min_api="2022-04-30-preview"
+        )
+        cmd_group.command(
+            "storage-container",
+            "message_endpoint_create_storage_container",
+            transform=EndpointUpdateResultTransform(self.cli_ctx)
+        )
+
+    with self.command_group(
+        'iot hub message-route', command_type=iothub_message_route_ops, is_preview=True
+    ) as cmd_group:
+        cmd_group.command(
+            'create', 'message_route_create', transform=RouteUpdateResultTransform(self.cli_ctx)
+        )
+        cmd_group.show_command('show', 'message_route_show')
+        cmd_group.command('list', 'message_route_list')
+        cmd_group.command(
+            'delete',
+            'message_route_delete',
+            transform=RouteUpdateResultTransform(self.cli_ctx),
+            confirmation=True
+        )
+        cmd_group.command(
+            'update', 'message_route_update', transform=RouteUpdateResultTransform(self.cli_ctx)
+        )
+        cmd_group.command('test', 'message_route_test')
```

Comment:
since the functionality here is different from normal test, this may need to be marked as preview